plugins/importexport/native/filter/NativeXmlPKPPublicationFilter.inc.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+/**
+ * @file plugins/importexport/native/filter/NativeXmlPKPPublicationFilter.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class NativeXmlPKPPublicationFilter
+ * @ingroup plugins_importexport_native
+ *
+ * @brief Base class that converts a Native XML document to a set of publications
+ */
+
+import('lib.pkp.plugins.importexport.native.filter.NativeImportFilter');
+
+class NativeXmlPKPPublicationFilter extends NativeImportFilter {
+	/**
+	 * Constructor
+	 * @param $filterGroup FilterGroup
+	 */
+	function __construct($filterGroup) {
+		$this->setDisplayName('Native XML publication import');
+		parent::__construct($filterGroup);
+	}
+
+
+	//
+	// Implement template methods from PersistableFilter
+	//
+	/**
+	 * @copydoc PersistableFilter::getClassName()
+	 */
+	function getClassName() {
+		return 'lib.pkp.plugins.importexport.native.filter.NativeXmlPKPPublicationFilter';
+	}
+
+
+	//
+	// Implement template methods from NativeImportFilter
+	//
+	/**
+	 * Return the plural element name
+	 * @return string
+	 */
+	function getPluralElementName() {
```

Comment:
(This is the same abstraction I mentioned in the OMP PR that I think can be removed.)